doc/amazon_sagemaker_debugger.rst

```diff
@@ -178,36 +178,36 @@ You can use the ``S3Downloader`` utility to view and download the debugging data
 Continuous analyses through rules
 =================================
 
-In addition to collecting the debugging data, Amazon SageMaker Debugger provides the capability for you to analyze it in a streaming fashion through "rules". A SageMaker Debugger 'rule' is a piece of code which encapsulates the logic for analyzing debugging data.
+In addition to collecting the debugging data, Amazon SageMaker Debugger provides the capability for you to analyze it in a streaming fashion using "rules". A SageMaker Debugger "rule" is a piece of code which encapsulates the logic for analyzing debugging data.
 
-SageMaker Debugger provides a set of built-in rules curated by data scientists and engineers at Amazon to identify common problems while training machine learning models. There is additional support for bringing in custom rule source codes for evaluation. In the following sections, you'll learn how to use either while training your model.
+SageMaker Debugger provides a set of built-in rules curated by data scientists and engineers at Amazon to identify common problems while training machine learning models. There is also support provided to use custom rule source codes for evaluation. In the following sections, you'll learn how to use both the built-in and custom rules while training your model.
```

Comment:
the original "for using" construct sounds better to me than "provided to use"

Reply:
Reverted to "for using".